web/satellite/src/components/account/AccountPaymentMethods.vue

```diff
@@ -45,11 +17,16 @@
 
 <script lang="ts">
     import { Component, Vue } from 'vue-property-decorator';
-    import Button from '@/components/common/Button.vue';
+    import CardComponent from '@/components/project/paymentMethods/CardComponent.vue';
 
     @Component({
+        computed: {
```

Comment:
we should change component decorator implementation to class component
@Component({omponents: {
            CardComponent,
        }
    })
export default class AccountPaymentMethods extends Vue {
    public get paymentMethods(): PaymentMethod[] {
                return this.$store.state.userPaymentsMethodsModule.userPaymentMethods;
            }
}